mmf/datasets/processors/bert_processors.py

```diff
@@ -141,17 +141,23 @@ def __call__(self, item):
 class BertTokenizer(MaskedTokenProcessor):
     def __init__(self, config, *args, **kwargs):
         super().__init__(config, *args, **kwargs)
-        self._probability = 0
```

Comment:
This option is useful to explicitly inform user that probability is zero. This is for readability purposes.